visualization_msgs/msg/Mesh.msg

```diff
@@ -0,0 +1,3 @@
+# Used to send raw mesh files.
+string format
```

Comment:
Expectations of this string should be documented. Is it a list of file types? 

For example see the Compressed image which lists the expected formats: https://github.com/ros2/common_interfaces/blob/d08d14194a108f8626fd05292961703df19374aa/sensor_msgs/msg/CompressedImage.msg#L10

This is important so that we know what standards to use and don't end up with both `dae` and `Collada` trying to express the same thing one in the abberviated suffix vs the full format name.

Reply:
Yeah, I'm kind of thinking we should just have one field be a file name, and if it's generated (i.e. no real file) then they can put something like `generated.dae`.

Reply:
Changed to `filename` and added a comment stating it should contain the file extension.